src/Dialogs/Airspace/dlgAirspaceWarnings.cpp

```
@@ -128,22 +134,44 @@ AirspaceWarningListWidget::UpdateButtons()
     ack_day_button->SetEnabled(false);
     enable_button->SetEnabled(false);
     radio_button->SetEnabled(false);
+    details_button->SetEnabled(false);
     return;
   }
 
   bool ack_expired, ack_day;
```

Cppcheck (2.20.0)

[error] Dereference of an invalid iterator

(derefInvalidIterator)